website/docs/d/is_security_groups.html.markdown

```diff
@@ -0,0 +1,115 @@
+
+Filter with VPC name
+data "ibm_is_security_groups" "example" {
+  vpc_name = "<Name of VPC>"
+}
+
+Filter with VPC ID
+data "ibm_is_security_groups" "example" {
+  vpc_id = "<VPC identifier>"
+}
+
+Filter with VPC CRN
+data "ibm_is_security_groups" "example" {
+  vpc_crn = "<VPC CRN>"
+}
+
+Filter with Resource Group ID
+data "ibm_is_security_groups" "example" {
+  resource_group="<Resource Group ID>"
+}
+```
+
+
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+- `id` - The unique identifier of the SecurityGroupCollection.
+- `vpc_name` - Filters the collection to resources in the VPC with the exact specified name
+- `vpc_id` - Filters the collection to resources in the VPC with the specified identifier
+- `vpc_crn` - Filters the collection to resources in the VPC with the specified CRN
+- `resource_group` -  Filters the collection to resources in the resource group with the specified identifier
+- `security_groups` - (Required, List) Collection of security groups.
+Nested scheme for **security_groups**:
+	- `created_at` - (Required, String) The date and time that this security group was created.
+	- `crn` - (Required, String) The security group's CRN.
+	- `href` - (Required, String) The security group's canonical URL.
+	- `id` - (Required, String) The unique identifier for this security group.
+	- `name` - (Required, String) The user-defined name for this security group. Names must be unique within the VPC the security group resides in.
+	- `network_interfaces` - (Deprecated, Required, List) The network interfaces for this security group.
+	Nested scheme for **network_interfaces**:
+		- `deleted` - (Optional, List) If present, this property indicates the referenced resource has been deleted and providessome supplementary information.
+		Nested scheme for **deleted**:
+			- `more_info` - (Required, String) Link to documentation about deleted resources.
+		- `href` - (Required, String) The URL for this network interface.
+		- `id` - (Required, String) The unique identifier for this network interface.
+		- `name` - (Required, String) The user-defined name for this network interface.
+		- `primary_ipv4_address` - (Required, String) The primary IPv4 address.If the address has not yet been selected, the value will be `0.0.0.0`.
+		- `resource_type` - (Required, String) The resource type.
+	- `resource_group` - (Required, List) The resource group for this security group.
+	Nested scheme for **resource_group**:
+		- `href` - (Required, String) The URL for this resource group.
+		- `id` - (Required, String) The unique identifier for this resource group.
+		- `name` - (Required, String) The user-defined name for this resource group.
+	- `rules` - (Required, List) The rules for this security group. If no rules exist, all traffic will be denied.
+	Nested scheme for **rules**:
+		- `code` - (Optional, Integer) The ICMP traffic code to allow.
```

Comment:
All required, optional needs to be removed here

Reply:
done.

Reply:
done.